src/components/views/rooms/EventTile.tsx

```diff
@@ -1071,6 +1072,18 @@ export default class EventTile extends React.Component<IProps, IState> {
         });
     };
 
+    /**
+     * In some cases we can't use shouldHideEvent() since whether or not we hide
+     * an event depends on other things that the event itself
+     * @returns {boolean} true if event should be hidden
+     */
+    private shouldHideEvent(): boolean {
+        // If the call was replaced we don't render anything since we render the other call
```

Comment:
I think what we'd actually want to do is merge the calls together so they appear as one long call, ie. take the start time as the time of the first call's invite and the hangup time as the ts of the other call's hangup event.

Reply:
I am not really sure what you have in mind... If we're talking about the time, I believe that should be handled by the js-sdk when replacing the call probably. Though I am not really sure how to intelligently handle that for past calls though since both calls happen at a similar time I am not even sure if this is necessary. 

Anyway, this feels like a different PR perhaps? Is there something else that should happen?

Reply:
Oh, as in we think this is already handled correctly by the js-sdk and we just have to display the right call and hide the other?

Reply:
No, I don't think it's handled in the js-sdk but I would make that a separate change and I also don't think it is that big of a deal though I might be missing something here :sweat_smile: